sherlock_project/resources/data.json

```diff
@@ -628,6 +628,13 @@
     "urlMain": "https://deviantart.com",
     "username_claimed": "blue"
   },
+  "DigitalSpy": {
+    "errorMsg": "The page you were looking for could not be found.",
+    "errorType": "message",
+    "url": "https://forums.digitalspy.com/profile/{}",
+    "urlMain": "https://forums.digitalspy.com/",
```

Comment:
It seems that DigitalSpy's username requirements are public:

> Username can only contain letters, numbers, underscores, and must be between 3 and 20 characters long.

Could you add a regexCheck to match this? Not sure if you've dealt with regex before. I can provide a pattern for you if need be.

Reply:
I would very much like to add a regexCheck if you'd provide me with how to proceed with it :)

Reply:
Simple enough for this one, actually. The word character class `\w` just so happens to expand to exactly those characters `[a-zA-Z0-9_]` (letters, numbers, underscores). Pattern here for 3-20 chars would just be `^\w{3,20}$` Would need to escape the backslash in this case due to Python stuff: `^\\w{3,20}$`. `^` for start of string and `$` for end of string.

So `"regexCheck": "^\\w{3,20}$"` should be it, really.

Prefilled regex101 link if curious about the pattern: https://regex101.com/r/4fLLj0/1